modules/ROOT/pages/migration-from-7x.adoc

```diff
@@ -3,9 +3,665 @@
 
-For additional details on {productname} {release-version} changes, see xref:8.0-release-notes.adoc#overview[{productname} {release-version} release notes].
\ No newline at end of file
+[IMPORTANT]
+.Breaking Changes Quick Reference
+====
+The following table summarizes all breaking changes in {productname} {release-version}. Each item links to detailed information further in the guide.
+
+Any items marked **"High"** level require immediate attention during migration.
+
+[cols="2,3,1",options="header"]
+|===
+|Breaking Change |Impact |Level
+|xref:license-key-system-update[License Key System]
+|Self-hosted deployments now require a new license key format and license key manager. Old keys are **not compatible**.
+|High
+
+|xref:dompurify-update-breaking-change[DOMPurify Update]
+|Sanitization is now stricter; content previously allowed may be stripped or altered.
+|High
+
+|xref:editor-selection-setcontent-deprecated[editor.selection.setContent]
+|Method deprecated. Use `editor.insertContent` instead.
+|Medium
+
+|xref:fire-method-deprecation[fire()]
+|Method deprecated. Use `dispatch()` for event handling.
+|Medium
+
+|xref:editor-documentbaseurl-removal[editor.documentBaseUrl]
+|Undocumented property removed. Use `editor.editorManager.documentBaseURI` instead.
+|Low
+
+|xref:skipfocus-consolidation[skipFocus and skip_focus]
+|Options consolidated to `skipFocus` in `ToggleToolbarDrawer`.
+|Low
+
+|xref:split-button-css-breaking-change[Split buttons]
+|Split button CSS classes and structure have changed.
+|Low
+|===
+====
+
+[[license-key-system-update]]
+=== Transition from Version 7 License Keys to Version {release-version} License Keys
+// #EPIC-192
+
+[IMPORTANT]
+====
+This section applies to self-hosted installations only. For cloud deployments, license key management is handled automatically.
+====
+
+{productname} {release-version} introduces an enhanced license key system that requires specific attention during migration.
+
+The complete licensing documentation xref:license-key.adoc[License Key Management] covers:
+
+* Detailed explanations of all license types (GPL, Commercial, GPL with Premium Features)
+* Time-based vs Version-locked license key differences
+* License states (Active, Grace Period, Expired, Invalid)
+* Deployment options (Cloud-only, Self-hosted, Hybrid)
+* Step-by-step configuration examples for each setup
+* Commercial License Key Manager setup and requirements
+* Troubleshooting and FAQ
+
+**Impact**: The new license key system introduces breaking changes that require updates to your configuration and code.
+
+**Key Migration Considerations**:
+
+* *License Key Format Change:* Version 7 keys are:
```

Comment:
Should we move “Version 7 keys are:” into the list item? It might make it clearer that the next points belong under `License Key Format Change`.